normalize_license_text/compare_normalized_files.py

```diff
@@ -1,8 +1,12 @@
 import sys
+import re
 
 """This file finally checks and matches the normalized files passed onto it 
 from the normalize.py file resulting from the output of NormalizeText class' return function.
 It returns the output of the result. """
 
 def CompareNormalizedFiles(a,b):
-    return a==b
+    if re.match(a,b):
```

Comment:
why have we moved to re.match() ?

Reply:
Actually there were some changes since the last evaluation, The template is converted to regex entirely hence the regex matching is required.

Reply:
okay 👍